Makefile

```diff
@@ -0,0 +1,17 @@
+RM=rm -rf
+
+.PHONY: all clean
+
+all:
+	yarn install
+	yarn workspace button build
+
+watch:
+	yarn workspace $(P) watch
+
+build:
+	yarn workspace $(P) build
+
+clean:
+	$(RM) yarn.lock node_modules
+	yarn workspace button clean
```

Comment:
If you go make a change to this clean command, I think also quickly add a `help` command at the top of the file and also set that as the default command for the makefile.